openstack/designate/requirements.yaml

```diff
@@ -2,9 +2,14 @@ dependencies:
   - name: mariadb 
     repository: file://../../common/mariadb
     version: 0.1.1
-  - name: memcached
-    repository: file://../../common/memcached
-    version: 0.0.1
   - name: rabbitmq
+    alias: rabbitmq_notifications
```

Comment:
this renames the existing (RPC-)rabbitmq. not good. We need an additional rabbitmq for notifications.

Reply:
fixed requirements with your help.